### DISCUSSION TITLE
Copy to system clipboard with Command+c

### DISCUSSION THREAD

**Discussion by backward-crazy-mage-puppy-36** [helix-editor/helix#5429] [Q&A]
Hi,
I would like to set Command+c to copy to system clipboard in all modes.
I have tried various keycap entries but was unable to get this working.
Can you help me out with the exact combination.

**Comment by the-mikedavis** [comment]
With the enhanced keyboard protocol (#4939) it is possible to add keybindings that use the super key (command for macos). However it doesn't look like it is possible to bind Super-c, I assume since the terminal program swallows that keypress. So I don't think it is possible to bind this in general.

**Comment by mkon** [comment]
How can one bind the super key? I am trying to bind Cmd-S to `:w`. I tried `Super-s`, `SUPER-s`, `Command-s` and `Cmd-s` but none of them are allowed modifiers when reloading the config (`helix 23.10 (f6021dd0)`).

Crossterm outputs these events:
```
Event: Key(KeyEvent { code: Char('s'), modifiers: KeyModifiers(SUPER), kind: Press, state: KeyEventState(0x0) })
Event: Key(KeyEvent { code: Char('s'), modifiers: KeyModifiers(SUPER), kind: Release, state: KeyEventState(0x0) })
```

It seems Super can only be used as a key and not a modifier?
https://github.com/helix-editor/helix/blob/a6ed104ea2787b523379895b7eb23998aae121f4/helix-view/src/input.rs#L395-L400

**Reply by the-mikedavis** [reply]
This is tracked by https://github.com/helix-editor/helix/issues/6582 now. It isn't implemented yet but I have a PR for it.
